### ISSUE TITLE
Random python issue in CNNTorch

### ISSUE THREAD

**Issue by jreps** [OHDSI/PatientLevelPrediction#66]
I'm getting this error in fold 3 [using covMod <- PatientLevelPrediction::setCNNTorch(nbfilters = 4, epochs = 1)]:
     File "C:/Users/jreps/Documents/R/win-library/3.5/PatientLevelPrediction/python/deepTorch.py", line 336, in forward
       out = self.bn(out)
     File "C:\Users\jreps\AppData\Local\Continuum\anaconda3\lib\site-packages\torch\nn\modules\module.py", line 477, in __call__
       result = self.forward(*input, **kwargs)
     File "C:\Users\jreps\AppData\Local\Continuum\anaconda3\lib\site-packages\torch\nn\modules\batchnorm.py", line 66, in forward
       exponenti

Some sort of issue in the anaconda batch prediction with pred_test1 = model.predict_proba(test)[:, 1] on line line 1089 in deepTorch.py.  Do you get this issue in non anaconda python?  I think it may be a random issue and will try again with slightly different data.

**Comment by jreps** [comment]
when I ran with different sized data the training all worked but then failed when saving the model.  It is a very odd thing, so I think the code is correct, but anaconda is not running it correctly? 

I want to release the new code as an official release (I've tested all the models and they work except the weird python issue with torch - but I think that is my python causing issues).  Can you test and if it works for you we can make an official release.

**Comment by xypan1232** [comment]
I will test it again. Before I tested it in Linux os and it works.

**Comment by xypan1232** [comment]
Hi @jreps , i saw you changed the "import cPickle" to "import _pickle as cPickle" in TorchUtils.py, does this only work on python 3.x , but not on python 2.7?

**Comment by xypan1232** [comment]
I removed this import, since cPickle is not used in this file. This is fixed in mi-erasmus and will push to OHDSI.

I tested  the torch CNN again, and it works on our server.
